src/com/content-games/games.js

```diff
@@ -166,55 +164,34 @@ export default class ContentGames extends Component {
 //        props.class.push("content-item-boxes");
 
 		var LoadMore = null;
-		const Games = [];
+		let Games = [];
```

Comment:
Actually isn't games only used once below, better remove here then and define it on that row as a `const`